### DISCUSSION TITLE
GTR MINI v1.0.0 emulator does not work

### DISCUSSION THREAD

**Discussion by XiaomaiTX** [zepp-health/zeppos-forum#126] [Developer Feedback]
GTR MINI v1.0.0 emulator does not work, as zeus dev connected to the emulator after the GTR MINI stuck
Here is my screen recording
[Screen Recording in community](https://zeppcommunity.slack.com/files/U050M52E8TE/F050SCX2KKM/666587369.mp4)

![1680075701765.jpg](https://user-images.githubusercontent.com/59403296/228551429-e7b49c84-1540-4d9c-be8c-a4861c86fb7a.jpg)

![1680075808886.jpg](https://user-images.githubusercontent.com/59403296/228551550-6676f4ab-1cc4-4f4b-9355-e44fcc4ac028.jpg)

**Comment by MyoungXUE** [comment]
Thank you for your feedback. This is a bug, and relevant colleagues are currently working on it.

**Reply by htoooth** [reply]
@XiaomaiTX  I suggest you use the button in the upper right corner, or if that doesn't work, reopen the software.
![Snipaste_2023-03-30_10-16-31](https://user-images.githubusercontent.com/1717023/228711039-22ea6f10-d5ee-42a0-a4b5-e61921b278f2.png)

**Reply by XiaomaiTX** [reply]
I tried it and the result is the same
